docs/reference/client.rst

```diff
@@ -60,17 +60,14 @@ Activity related methods
    :toctree: api/
 
    Client.get_activity
```

Comment:
Removed deprecated methods. the API docs themselves will need a bit more cleanup but this just ensures things build without errors